pkg/db/db_local/running_info.go

```diff
@@ -17,33 +17,26 @@ const RunningDBStructVersion = 20220411
 
 // RunningDBInstanceInfo contains data about the running process and it's credentials
 type RunningDBInstanceInfo struct {
-	Pid           int               `json:"pid"`
-	Port          int               `json:"port"`
-	Listen        []string          `json:"listen"`
-	ListenType    StartListenType   `json:"listen_type"`
-	Invoker       constants.Invoker `json:"invoker"`
-	Password      string            `json:"password"`
-	User          string            `json:"user"`
-	Database      string            `json:"database"`
-	StructVersion int64             `json:"struct_version"`
+	Pid             int               `json:"pid"`
+	ListenAddresses string            `json:"listen"` // FIXME: this may be a breaking change, not sure if it really matters
```

Comment:
Not sure if this is a breaking change, or if it really matters.

Reply:
In the current state of this PR, it still holds all the addresses, just not as an array of strings, but as a string containing comma separated addresses as specified in configuration. Splitting is done a later stage only when needed,e.g. when we only need the first address.

Can revert that though.

Reply:
@pdecat changing the structure of the file is probably not a good idea unless there's a very compelling reason to do so. I think we should maintain the array type for `listen`. 

For `local` and `network` this should contain all the interfaces that it bound to. For `network` this will effectively be all the interfaces in the system.

For cases where hosts are mentioned, it should be stored as an array where each element is a host/address

Reply:
@pdecat did you get a chance to look at this one?

Reply:
Not yet, I hope to find time to get to it in the next two weeks.

Reply:
Done.